openapitools.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think this file is here by mistake. You may have generated it when trying to use the `npm` package, but if I run the docker generator I don't get this file.